.github/workflows/test.yaml

```diff
@@ -28,28 +59,17 @@ jobs:
           pip install -r requirements.txt
           pip install -r test-requirements.txt
 
-      - name: Test
-        env:
-          BW_ACCOUNT_ID: ${{ secrets.BW_ACCOUNT_ID }}
-          BW_USERNAME: ${{ secrets.BW_USERNAME }}
-          BW_PASSWORD: ${{ secrets.BW_PASSWORD }}
-          BW_USERNAME_FORBIDDEN: ${{ secrets.BW_USERNAME_FORBIDDEN }}
-          BW_PASSWORD_FORBIDDEN: ${{ secrets.BW_PASSWORD_FORBIDDEN }}
-          BW_VOICE_APPLICATION_ID: ${{ secrets.BW_VOICE_APPLICATION_ID }}
-          BW_MESSAGING_APPLICATION_ID: ${{ secrets.BW_MESSAGING_APPLICATION_ID }}
-          BW_NUMBER: ${{ secrets.BW_NUMBER }}
-          USER_NUMBER: ${{ secrets.USER_NUMBER }}
-          VZW_NUMBER: ${{ secrets.VZW_NUMBER }}
-          ATT_NUMBER: ${{ secrets.ATT_NUMBER }}
-          T_MOBILE_NUMBER: ${{ secrets.T_MOBILE_NUMBER }}
-          BASE_CALLBACK_URL: ${{ secrets.BASE_CALLBACK_URL }}
-          PYTHON_VERSION: ${{ matrix.python-version }}
-          OPERATING_SYSTEM: ${{ matrix.os }}
-          MANTECA_ACTIVE_NUMBER: ${{ secrets.MANTECA_ACTIVE_NUMBER }}
-          MANTECA_IDLE_NUMBER: ${{ secrets.MANTECA_IDLE_NUMBER }}
-          MANTECA_BASE_URL: ${{ secrets.MANTECA_BASE_URL }}
-          MANTECA_APPLICATION_ID: ${{ secrets.MANTECA_APPLICATION_ID }}
-        run: pytest
+      - name: Test at Debug Level
+        if: ${{ inputs.logLevel == 'DEBUG' }}
+        run: |
+          echo "Log level: DEBUG"
+          pytest -v --log-cli-level=DEBUG
+
+      - name: Test at Warning Level
+        if: ${{( inputs.logLevel == null) || ( inputs.logLevel  == 'WARNING') }}
```

Comment:
not super familiar with default values in GHA, but is the `== null` necessary? Will the value become `WARNING` if no value is passed in?

Reply:
No that's the default that it's on when you manually select the workflow dispatch. In the actions that ran it is coming through as null when it gets run normally on a PR commit so we need that in there.